### ISSUE TITLE
please don't call changelog to release notes

### ISSUE THREAD

**Issue by carandraug** [olivierlacan/keep-a-changelog#256]
Both the GNU project and Debian make a distinction between release notes or news, and a changelog but their definition clashes with keep-a-changelog.

[GNU standards](https://www.gnu.org/prep/standards/standards.html#NEWS-File) roughly defines NEWS as the file that contains a list of user-visible changes worth mentioning and changelog as the file that describe all the changes made to program source files.

[Debian policy](https://www.debian.org/doc/debian-policy/ch-docs.html#changelog-files-and-release-notes) also distinguishes NEWS and changelog. It has no definition for changelog, only for NEWS which is defined as "containing a summary of changes between upstream releases intended for end users of the package". However, the [discussion](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=459427) that led to that point on Debian policy is a bit clearer and have changelog as "source-level change list"

My interpretation of keep-a-changelog is that it's advocating for a release notes / NEWS type of file. Thank you. However, it's a bit confusing that it suggest calling such file changelog, the opposite of existing standards and policy.

**Comment by while0pass** [comment]
I suppose, there are at least 3 levels of changelogs, each one sequentially expands its target audience and decreases the amount of detail:

1) Commit log, targeted at the developers of the product;
2) Changelog, additionally targeted at the third-party developers (using the product as a dependency for their products) and advanced users of the product;
3) Release notes or News, targeted at all users and developers of the product.

The difference between (1) and (2) is more about systemizing the changes, and between (2) and (3) -- about summarizing and synopsizing. Keep-a-changelog looks like (2), I suppose.

**Comment by olivierlacan** [comment]
We've debated this issue to death in the past. I appreciate your attempt at clarification but GNU and Debian are not divine law. Their conception of a changelog is flawed and I've explained (both inside Keep a Changelog itself and in discussions here) that I don't think the NEWS naming is as clear and appropriate for what I (and many others in the open source community) think a changelog should offer to end users.

I really like @while0pass's summary of the differences between all layers of communication on a project. It reminds me of the [OSI model][1] a bit. I've been wrestling for a while with the border between changelogs and release notes and I think there's a level of editorial newsworthiness required for changelog items to be included within release notes that is a bit or much higher. As @while0pass astutely pointed out, the intended audience for changelogs is often much more technical than the usual intended audience for release notes. Even if we're talking about developer tooling. 

[1]: https://en.wikipedia.org/wiki/OSI_model
